token-lending/program/src/instruction.rs

```diff
@@ -365,16 +398,26 @@ impl LendingInstruction {
                 buf.extend_from_slice(&amount.to_le_bytes());
                 buf.extend_from_slice(&amount_type.to_u8().unwrap().to_le_bytes());
             }
-            Self::RepayReserveLiquidity { liquidity_amount } => {
+            Self::MarginBorrowReserveLiquidity {
+                collateral_amount,
+                loan_amount,
+                amount_type,
+            } => {
                 buf.push(6);
+                buf.extend_from_slice(&collateral_amount.to_le_bytes());
+                buf.extend_from_slice(&loan_amount.to_le_bytes());
+                buf.extend_from_slice(&amount_type.to_u8().unwrap().to_le_bytes());
+            }
+            Self::RepayReserveLiquidity { liquidity_amount } => {
+                buf.push(7);
                 buf.extend_from_slice(&liquidity_amount.to_le_bytes());
             }
             Self::LiquidateObligation { liquidity_amount } => {
-                buf.push(7);
+                buf.push(8);
                 buf.extend_from_slice(&liquidity_amount.to_le_bytes());
             }
             Self::AccrueReserveInterest => {
-                buf.push(8);
+                buf.push(9);
```

Comment:
changing the other codes will affect the existing functionality through the UI:
https://github.com/solana-labs/oyster-lending/blob/main/src/models/lending/lending.ts

Perhaps just make the margin borrow a new code, and leave the others to allow the UI to work as is for other calls?

Reply:
oh okay good call